python-main.js

```diff
@@ -24,19 +24,43 @@ is attached to the div with the referenced id.
 */
 function pythonEditor(id) {
     'use strict';
+    ace.config.set('basePath', '/ace');
```

Comment:
Is this still necessary after the other test fixes? Can't it be removed?

Reply:
Tests pass, but we see the following error on the console:

```
  console.error ../ace/ace.js:1
    Unable to infer path to ace from script src, use ace.config.set('basePath', 'path') to enable dynamic loading of modes and themes or with webpack use ace/webpack-resolver
```

Reply:
If it's needed only for the tests then we can probably move that to the test_setup.js file?

Reply:
Moved in https://github.com/bbcmicrobit/PythonEditor/pull/128/commits/ed540bd6294b7657b61acd6e97f5ab067a61d483